File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/UserInfoPresenter.java
package bobrchess.of.by.belaruschess.presenter;

import bobrchess.of.by.belaruschess.view.activity.impl.UserInfoActivity;

/**
 * Created by Igor on 12.04.2018.
 */

public interface UserInfoPresenter extends BasePresenter{
    void attachView(UserInfoActivity activity);

    void loadUserTournaments();
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/UserInfoPresenter.kt
package bobrchess.of.by.belaruschess.presenter

import bobrchess.of.by.belaruschess.outdated.MainOldActivity

/**
 * Created by Igor on 12.04.2018.
 */

interface UserInfoPresenter : BasePresenter {
    fun attachView(activity: MainOldActivity)

    fun loadUserTournaments()
}
